### PR TITLE
Add <meta charset="UTF-8">

### PR DESCRIPTION
**Pull request by yukoba**
I added ```<meta charset="UTF-8">``` to page_template.html for non ascii characters support.

- [x] Tick to sign-off your agreement to the [Developer Certificate of Origin (DCO) 1.1](https://developercertificate.org)

### CODE REVIEW

File: jsplot/src/main/java/tech/tablesaw/plotly/Plot.java [hunk 1]
```diff
@@ -28,8 +31,8 @@ public static void show(Figure figure, String divName, File outputFile) {
         String output = page.asJavascript();
 
         try {
-            try (FileWriter fileWriter = new FileWriter(outputFile)) {
-                fileWriter.write(output);
+            try (Writer writer = new OutputStreamWriter(new FileOutputStream(outputFile), StandardCharsets.UTF_8)) {
```
Comment (benmccann) [inline jsplot/src/main/java/tech/tablesaw/plotly/Plot.java]: why are you changing this?

**Comment by benmccann** [comment]
Adding the `meta` tag and `DOCTYPE` tag are small and related changes and would be better to do in a single PR

File: jsplot/src/main/java/tech/tablesaw/plotly/Plot.java (hunk 1 above)
Comment (yukoba) [reply jsplot/src/main/java/tech/tablesaw/plotly/Plot.java]: FileWriter uses ```file.encoding``` encoding and if there is no ```file.encoding``` encoding, Java uses UTF-8.
Sometimes ```file.encoding``` is not UTF-8 and I forced to use UTF-8.

**Comment by yukoba** [comment]
Rebased and merged two git branches.

File: jsplot/src/main/java/tech/tablesaw/plotly/Plot.java (hunk 1 above)
Comment (benmccann) [reply jsplot/src/main/java/tech/tablesaw/plotly/Plot.java]: Won't this still use UTF-8 encoding since it's passing `StandardCharsets.UTF_8`?

Comment (benmccann) [reply jsplot/src/main/java/tech/tablesaw/plotly/Plot.java]: Oh, I misunderstood you comment. You want to force it to use UTF-8. That makes sense. Thanks

Comment (yukoba) [reply jsplot/src/main/java/tech/tablesaw/plotly/Plot.java]: Yes. I want to force it to use UTF-8.

**Comment by benmccann** [comment]
Thanks!

**Pull request merged**